### DISCUSSION TITLE
CSS Themes

### DISCUSSION THREAD

**Discussion by HyperCube90** [Lissy93/dashy#879] [Q&A]
Hello, I hope I'm right here. I have a few questions and I hope you can help me.

I use Dashy and find the dashboard really good, but I would like to create my own CSS or edit an existing Lissy CSS to e.g. customize widgets, shapes, groups and much more.

I tried to put the CSS / SCSS file in the same folder where the config file is located. A CSS name is displayed under Theme, but I think that's only because I wrote the name of the CSS / SCSS (color-palette) in the config file.
But nothing changed after selecting the theme 'color-palette'

I actually followed the instructions in the wiki, unfortunately without success.

Does anyone have an example for me with pictures of the file and folder structure?

I run Dashy on a Docker container based on unraid.

I really hope that someone can help me and if I'm in the wrong area please let me know I'm pretty overwhelmed and new when it comes to github.

Thanks :)

**Comment by shimizurei** [comment]
Have you read this page?: https://dashy.to/docs/theming
